doc/nmstatectl.8

```diff
@@ -0,0 +1,89 @@
+.\" Manpage for nmstatectl.
+.TH man 8 "08 Oct 2019" "0.0.8" "nmstatectl man page"
+.SH NAME
+nmstatectl \- A nmstate command line tool
+.SH SYNOPSIS
+.B nmstatectl show \fR[<\fIinterface_name\fR>] [\fB--json\fR]
+.br
+.B nmstatectl set \fR<\fIstate_file_path\fR> [\fIOPTIONS\fR]
+.br
+.B nmstatectl edit \fR[<\fIinterface_names\fR>] [\fIOPTIONS\fR]
+.br
+.B nmstatectl rollback \fR<\fIcheckpoint_path\fR>
+.br
+.B nmstatectl commit \fR<\fIcheckpoint_path\fR>
+.SH DESCRIPTION
+.B nmstatectl\fR is created for users who want to try out nmstate without using
+\fIlibnmstate\fR.
+.PP
+.B show
+.RS
+Query the current network state. \fIYAML\fR is the default output format. Use
+the \fB--json\fR argument to change the output format to \fIJSON\fR. To limit
+the output state to include certain interfaces only, please specify the
+interface name. Please be advised, global config like DNS will be included.
+.PP
+For multiple interface names, use comma to separate them. You can also use
+regular expression patterns for interface names:
```

Comment:
The pattern syntax does not conform to regular expressions. I would just remove the words `regular expression`. Regular expressions would be the following:

`.*`
`.`
`[seq]`
`[^seq]`